third_party/kythe/java/com/google/devtools/kythe/extractors/shared/CAMPFIRE

```diff
@@ -18,5 +18,21 @@
         "//third_party/protobuf:protobuf_jar"
       ]
     }
+  },
+  {
+    "name": "file_vnames",
```

Comment:
This target isn't used by Shipshape.  Also, this target is the only dependent on //third_party/re2j.

./campfire query --print_names 'deps("//shipshape/...")' | grep third_party/kythe